docs/jwst/master_background/description.rst

```diff
@@ -0,0 +1,50 @@
+Description
+===========
+The master spectroscopic background subtraction step subtracts background signal from
+2-D spectroscopic data using a 1-D master background spectrum. The 1-D master background
+spectrum is computed from one or more input exposures, or can alternatively be supplied
+by the user. The 1-D background spectrum - flux versus wavelength - is projected into the
+2-D space of source data based on the wavelength of each pixel in the 2-D data. The resulting
+2-D background signal is then subtracted directly from the 2-D source data.
+
+Upon successful completion of the step, the S_MSBSUB keyword is set to 'COMPLETE' in the
+output product. The background-subtracted results are returned as a new data model, leaving
+the input model unchanged.
+
+Creating the 1-D master background spectrum
+-------------------------------------------
+The 1-D master background spectrum is created by combining the data contained in the
+:ref:`x1d <x1d>` products created by the :ref:`calwebb_spec2 <calwebb_spec2>` pipeline. The
+collection of exposure products to be used as background is designated in the input ASN file,
+by setting 'exptype=background' for the appropriate product members. The background members
+could be exposures obtained of dedicated background targets or could be a collection of
+on-target exposures for a point-source observed in a nod pattern (e.g. MIRI LRS fixed-slit
+2-point nods).
+
+The flux versus wavelength data in the :ref:`x1d <x1d>` products for the designated background members
+is combined using the :ref:`combine_1d <combine_1d_step>` step to produce the 1-D master
+background spectrum.
+
+[Need a brief description here of how the multiple spectra are resampled/combined in
+wavelength space to form the final MSB spectrum, once that's been decided.]
+
+Subtracting the master background
+---------------------------------
+The 1-D master background spectrum is projected into the 2-D space of each source data instance
```

Comment:
`is interpolated by wavelength at each pixel to the 2D source spectrum  and subtracted`